GliaWidgets/Sources/Interactor/Interactor.swift

```diff
@@ -289,7 +290,13 @@ extension Interactor: CoreSdkClient.Interactable {
     }
 
     var onLiveToSecureConversationsEngagementTransferring: CoreSdkClient.EngagementTransferringBlock {
-        return {}
+        return { [weak self, environment] in
+            environment.log.prefixed(Self.self).info(
```

Comment:
Just want to confirm if this newly introduced log is added to logging table as suggested in [guide](https://glia.atlassian.net/wiki/x/awD31Q).

Reply:
This log is added to the Google sheet